test/testdata/resolver/redefined_abstract.rb

```diff
@@ -0,0 +1,28 @@
+# typed: true
+
+module IFoo
```

Comment:
Given that the report concerns what appears to be a multi-file test, it seems like we also ought to have a multi-file test for this?

Reply:
sure, i've added one.